src/BottomTab.js

```diff
@@ -0,0 +1,77 @@
+import './BottomTab.css'
+import {AppModes, TasksShowing} from "./App";
+import {useState} from "react";
+
+function BottomTab(props) {
+    const [idsToBeDeleted, setIDsToBeDeleted] = useState(null);
```

Comment:
You can store this state, but might be simpler to store the minimal state needed:
deleteSelected could contain one of three values: null (nothing to be deleted), "All", or "Completed".

Then, you can still put up the confirmation dialog.

General rule is don't store state that can be recomputed (like the ids to be deleted), just recompute things as necessary.